tor/src/main/scala/org/bitcoins/tor/client/TorClient.scala

```diff
@@ -182,7 +182,9 @@ object TorClient extends Logging {
         "linux_64/libssl.so.1.1",
         "linux_64/libevent-2.1.so.7",
         "linux_64/libcrypto.so.1.1",
-        "linux_64/libstdc++/libstdc++.so.6"
+        "linux_64/libstdc++/libstdc++.so.6",
+        "linux_64/PluggableTransports/obfs4proxy",
```

Comment:
According to ben's docs these should not be copied 

https://github.com/bitcoin-s/bitcoin-s/tree/master/tor/src/main/resources#updating-executables